mne/annotations.py

```diff
@@ -120,13 +119,14 @@ def _onset_to_seconds(raw, onset):
         meas_date = 0
     elif not np.isscalar(meas_date):
         if len(meas_date) > 1:
-            meas_date[1] /= 1e6
-        meas_date = meas_date.sum()
+            meas_date = float(meas_date[0]) + meas_date[1] / 1000000.
```

Comment:
you shouldn't need this `float` cast it will happen automatically because the right operand of `+` is a float